packages/shift-next/src/pages/static-page.js

```diff
@@ -57,12 +73,13 @@ class StaticPage extends Component {
     }
   }
 
-  renderPageTitle (title) {
+  renderPageHead (canonicalPath, title) {
     const homepage = title === 'Homepage'
```

Comment:
@Azfletch might be worth using `const homepage = canonicalPath === '/pages/homepage'` instead, as this is what's used over here: https://github.com/shiftcommerce/reference-site/blob/master/packages/reference/server/server.js#L72

Reply:
👍